dask/config.py

```diff
@@ -526,7 +558,7 @@ def update_defaults(new, config=config, defaults=defaults):
     update(config, new, priority="old")
 
 
-def expand_environment_variables(config):
+def expand_environment_variables(config: Any) -> Any:
```

Comment:
I don't want to break backwards compatibility, so the `Any` branch should stay, but I wish it weren't here.